### PR TITLE
feat(j-s): feature flag modify ruling button

### PR DESCRIPTION
**Pull request by Arnarkari93**
[Asana - feature flag Leiðrétta úrskurð](https://app.asana.com/0/1199153462262248/1202360581950457/f)
## What

- But Leiðrétta button under feature flag

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverviewForm.tsx [hunk 1]
```diff
@@ -390,6 +394,7 @@ const SignedVerdictOverviewForm: React.FC<Props> = (props) => {
               ) : user?.role === UserRole.JUDGE ||
                 user?.role === UserRole.REGISTRAR ? (
                 <Button
+                  variant="ghost"
```
Comment (Arnarkari93) [inline apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverviewForm.tsx:397]: sneek in making both buttons ghoast 🤫

File: charts/judicial-system/values.prod.yaml [hunk 2]
```diff
@@ -19,7 +19,7 @@ judicial-system-api:
     AUTH_AUDIENCE: 'rettarvorslugatt.island.is'
     BACKEND_URL: 'http://web-judicial-system-backend'
     CONTENTFUL_HOST: 'cdn.contentful.com'
-    HIDDEN_FEATURES: ''
```
Comment (gudjong) [inline charts/judicial-system/values.prod.yaml:22]: Consider also hiding on staging, so the hiding can be tested there.

Comment (gudjong) [reply charts/judicial-system/values.prod.yaml:22]: Note, that the yaml files should not be modified directly. The judicial-system-api.ts file in the infra folder of the judicial-system-api application should be modified and the yaml files generated from it.

**Review by gudjong — CHANGES_REQUESTED** [review]

File: charts/judicial-system/values.prod.yaml (hunk 2 above)
Comment (Arnarkari93) [reply charts/judicial-system/values.prod.yaml:22]: Addressed here: https://github.com/island-is/island.is/pull/7545/commits/847862f2d6978e1e5be581917b13cbf7628580b5

**Review by gudjong — APPROVED** [review]

**Review by Snojo — APPROVED** [review]
Looks good on DevOps side

**Pull request merged**